modules/programs/zsh.nix

```diff
@@ -360,15 +368,39 @@ in
     })
 
     (mkIf (cfg.profileExtra != "") {
-      home.file."${relToDotDir ".zprofile"}".text = cfg.profileExtra;
+      home.file."${relToDotDir ".zprofile"}".text =
```

Comment:
Could try to use something like
```nix
(mkIf cfg.prezto.enable {
  programs.zsh.profileExtra = builtins.readFile "${pkgs.zsh-prezto}/runcoms/zprofile";
})
```
maybe
```nix
mkIf cfg.prezto.enable {
  programs.zsh.profileExtra = mkBefore (builtins.readFile "${pkgs.zsh-prezto}/runcoms/zprofile");
}
```
if the prezto code need to be early in the profile.

If it works then it should be possible to remove the `(mkIf (cfg.prezto.enable && cfg.profileExtra == "") {` below.

Reply:
Actually, perhaps simply put
```
programs.zsh.profileExtra = builtins.readFile "${pkgs.zsh-prezto}/runcoms/zprofile";
```
in the `prezto.nix` file?